src/components/TagWallFilter/TagWallFilter.stories.js

```diff
@@ -33,58 +31,34 @@ const buttons = {
   },
 };
 
-const itemsReducer = (items, item) => {
-  items[item.id] = item;
-  return [items, item];
-};
-const filterItems = selectedItemsMap => ([items, item]) => {
-  if (item.id in selectedItemsMap) {
-    delete items[item.id];
-  }
-  return [items, item];
-};
-const selectFirst = ([items]) => items;
+storiesOf(patterns('TagWallFilter'), module)
+  .add(`TagWall with Filter list for multiselect`, () => {
+    const selectedItems = object('selectedItems', [items[0]]);
```

Comment:
I removed a lot of this complicated functional stuff here, as I think as a user of the storybook - its much easier to understand the API of the component if we just use a hard coded array.

Perhaps I regressed something, let me know